k8s/contiv-vpp-arm64.yaml

```diff
@@ -26,7 +26,7 @@ data:
     UseTAPInterfaces: true
     TAPInterfaceVersion: 2
     NatExternalTraffic: true
-    MTUSize: 1450
+    MTUSize: 1500
```

Comment:
Do not commit changes in yaml file without changing the helm template - it should reflect helm generation with default values